module/modmanager/options/options.go

```diff
@@ -11,6 +11,12 @@ import (
 type Options struct {
 	UntrustedEnv bool
 
+	// Root of Viam server configuration. This is generally in either /opt/viam or ~/.viam
```

Comment:
```suggestion
	// Root of Viam server configuration. This is generally either /opt/viam or ~/.viam
```
Just for my own understanding, this is expected to literally be `/opt/viam` or `~/.viam`, not a directory within those directories, correct?

Reply:
Correct -- clarified in comment